controls/cis_debian12.yml

```diff
@@ -0,0 +1,3110 @@
+
+  - id: 2.1.18
+    title: Ensure web server services are not in use (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    rules:
+      - package_httpd_removed
+      - package_nginx_removed
+      - service_httpd_disabled
+      - service_nginx_disabled
+    status: automated
+
+  - id: 2.1.19
+    title: Ensure xinetd services are not in use (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    rules:
+      - package_xinetd_removed
+      - service_xinetd_disabled
+    status: automated
+
+  - id: 2.1.20
+    title: Ensure X window server services are not in use (Automated)
+    levels:
+      - l2_server
+    rules:
+      - package_xorg-x11-server-common_removed
+    status: automated
+
+  - id: 2.1.21
+    title: Ensure mail transfer agent is configured for local-only mode (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    rules:
+      - has_nonlocal_mta
+      - var_postfix_inet_interfaces=loopback-only
+      - postfix_network_listening_disabled
+    status: automated
+
+  - id: 2.1.22
+    title: Ensure only approved services are listening on a network interface (Manual)
+    levels:
+      - l1_server
+      - l1_workstation
+    status: manual
+
+  - id: 2.2.1
+    title: Ensure NIS Client is not installed (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    rules:
+      - package_nis_removed
+    status: automated
+
+  - id: 2.2.2
+    title: Ensure rsh client is not installed (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    rules:
+      - package_rsh_removed
+    status: automated
+
+  - id: 2.2.3
+    title: Ensure talk client is not installed (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    rules:
+      - package_talk_removed
+    status: automated
+
+  - id: 2.2.4
+    title: Ensure telnet client is not installed (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    rules:
+      - package_telnet_removed
+    status: automated
+
+  - id: 2.2.5
+    title: Ensure ldap client is not installed (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    rules:
+      - package_openldap-clients_removed
+    status: automated
+
+  - id: 2.2.6
+    title: Ensure ftp client is not installed (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    rules:
+      - package_ftp_removed
+      - package_tnftp_removed
+    status: automated
+
+  - id: 2.3.1.1
+    title: Ensure a single time synchronization daemon is in use (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    rules:
+      - var_timesync_service=systemd-timesyncd
+      # Choose chrony
+      - package_chrony_installed
+      # - package_timesyncd_removed # Not written in CIS documentation
+      - service_timesyncd_disable 
+      ##
+      # Choose systemd-timesyncd
+      # - package_timesyncd_installed # Not written in CIS documentation
+      - package_chrony_removed
```

Comment:
FTBFS here.

```
ValueError: Control cis_debian12:2.3.1.1 contains nonexisting rule(s) package_chrony_removed, service_timesyncd_disable
```

Only existing rules can be added to cfile. The rule package_chrony_removed really doesn't exist, the service_timesyncd_disable should be service_timesyncd_disabled.